### PR TITLE
update brokering folder google drive location in Wrangling docs

### PR DESCRIPTION
**Pull request by amnonkhen**

### CODE REVIEW

File: docs/ebi-wrangler-onboarding.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ You have unlimited storage in Google Drive. Use it for storing all the documents
 
 Google Docs has most of the important documents that are used by the team on a regular basis. We don't expect you to know all of them the first day, so ask for a link to the document if you can't find it. Searching is useful for finding documents (if you know what you're looking for) but if there's anything you can't find do ask.
 
-A good place to start is by checking out the `Brokering` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks.
+A good place to start is by checking out the `[Data Lifecycle and Brokering Project](https://drive.google.com/drive/u/0/folders/165VFrdVBHRciibpflBoxtmGDqV04VCFw)` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks.
```
Comment (Wkt8) [inline docs/ebi-wrangler-onboarding.md:195]: I think the appropriate link is actually: https://drive.google.com/drive/folders/1y_6QcB_laGdfKAUklNaC0i2SPV7zwGJI

Which is the 'Brokering' folder. Hence, would recommend leaving it as 'Brokering' and adding the new url.

**Review by Wkt8 — CHANGES_REQUESTED** [review]

File: docs/ebi-wrangler-onboarding.md (hunk 1 above)
Comment (ESapenaVentura) [reply docs/ebi-wrangler-onboarding.md:195]: `PROJECTS - ` + IN PROGRESS, FINISHED, etc

**Review by ESapenaVentura — APPROVED** [review]
Other than the change requested by Wei, LGTM

File: docs/ebi-wrangler-onboarding.md (hunk 1 above)
Comment (amnonkhen) [reply docs/ebi-wrangler-onboarding.md:195]:
```suggestion
A good place to start is by checking out the `[Data Lifecycle and Brokering Project](https://drive.google.com/drive/folders/1y_6QcB_laGdfKAUklNaC0i2SPV7zwGJI)` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks.
```

Comment (amnonkhen) [inline docs/ebi-wrangler-onboarding.md]:
```suggestion
A good place to start is by checking out the `[Data Lifecycle and Brokering Project](https://drive.google.com/drive/u/0/folders/165VFrdVBHRciibpflBoxtmGDqV04VCFw)` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks. The spreadsheets are in: `PROJECTS - [IN PROGRESS|FINISHED]`, etc
```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
Just added a small comment, other than that LGTM 👍

File: docs/ebi-wrangler-onboarding.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ You have unlimited storage in Google Drive. Use it for storing all the documents
 
 Google Docs has most of the important documents that are used by the team on a regular basis. We don't expect you to know all of them the first day, so ask for a link to the document if you can't find it. Searching is useful for finding documents (if you know what you're looking for) but if there's anything you can't find do ask.
 
-A good place to start is by checking out the `Brokering` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks.
+A good place to start is by checking out the `[Brokering](https://drive.google.com/drive/folders/1y_6QcB_laGdfKAUklNaC0i2SPV7zwGJI)` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks.
```
Comment (amnonkhen) [inline docs/ebi-wrangler-onboarding.md:195]:
```suggestion
A good place to start is by checking out the `[Brokering](https://drive.google.com/drive/folders/1y_6QcB_laGdfKAUklNaC0i2SPV7zwGJI)` folder (search is your friend) which is mainly where we store documents related to our ongoing wrangling tasks. The spreadsheets are in: `PROJECTS - [IN PROGRESS|FINISHED]`, etc
```

**Pull request merged**